cluster/prod/knative/summarizer.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: testgrid-summarizer
+  namespace: knative-testgrid
+  labels:
+    component: summarizer
+    app: testgrid
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: testgrid
+      component: summarizer
+  template:
+    metadata:
+      labels:
+        component: summarizer
+        app: testgrid
+    spec:
+      serviceAccountName: summarizer
+      containers:
+      - name: summarizer
+        image: gcr.io/k8s-testgrid/summarizer:v20210201-v0.0.45-2-ga6373ea
+        args:
+        - --config=gs://knative-own-testgrid/config
```

Comment:
is gs://knative-testgrid unavailable?

Reply:
it's available, but contains config that's consumed by testgrid.knative.dev, which is still functioning I think. Keeping it for legacy reason, I can perform the transition some time later